adapters/beintoo/beintoo.go

```diff
@@ -0,0 +1,256 @@
+package beintoo
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type BeintooAdapter struct {
+	endpoint string
+	testing  bool
+}
+
+func buildEndpoint(endpoint string, testing bool, timeout int64) string {
+	
+	if timeout == 0 {
+		timeout = 1000
+	}
+	
+	uriObj, _ := url.Parse(endpoint)
+	parameters := url.Values{}
+	
+	if testing {
+		
+		parameters.Add("t", "1000")
+		parameters.Add("ts", "2060541160")
+		
+	} else {
+	
+	parameters.Add("t", strconv.FormatInt(timeout, 10))
+	parameters.Add("ts", strconv.FormatInt(time.Now().Unix(), 10))
```

Comment:
Can we add one more test case to cover the `else` branch?
![image](https://user-images.githubusercontent.com/15863840/81324398-ec12b600-9064-11ea-826f-9afc61bb1036.png)